src/react-sortable-tree.js

```diff
@@ -471,6 +474,7 @@ ReactSortableTree.propTypes = {
     // This is an advanced option for complete customization of the appearance.
     // It is best to copy the component in `node-renderer-default.js` to use as a base, and customize as needed.
     nodeContentRenderer: PropTypes.any,
+    contentRenderer: PropTypes.func.isRequired,
```

Comment:
Could you add a comment explaining the purpose here? The same explanation should go in the README.md as well.